core/src/main/java/com/graphhopper/util/Parameters.java

```diff
@@ -191,9 +191,9 @@
      */
     public static final class DETAILS {
 
-        private static final String DETAILS_PREFIX = "details.";
```

Comment:
> Ok, I think we should keep the variable 'DETAILS_PREFIX' as we do similar for the others.

Do I understand you right, that I should revert the change then? Or just change `path_details` to `details`? So that a request might look like `details=average_speed`. I updated this PR use `details=average_speed`.

> But the most important thing is: if we name the parameter x then the json entry should be also x

I agree 100%, I missed that we have the return type named as details.

> Also regarding the debugging I would just use debug=true

This would conflict with the debug parameter of the API. There we also got a debug parameter that formats the json output. Not that this would be an issue, just wanted to mention it. Changed this as well.